src/DotLiquid.Tests/Tags/StandardTagTests.cs

```diff
@@ -182,7 +182,7 @@ public void TestLimiting()
         public void TestDynamicVariableLimiting()
         {
             Hash assigns = Hash.FromAnonymousObject(new { array = new[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 0 } });
-            assigns["limit"] = 2;
+            assigns["limit"] = 2L;

```

Comment:
32 bit limit makes a lot more sense here due to compute cost of a 64 bit boundary.